angular/test/protractor/group_spec.coffee

```diff
@@ -12,6 +12,26 @@ describe 'Group Page', ->
         page.expectElement('link[rel=prev]')
         page.expectElement('link[rel=next]')
 
+      it 'should allow you to join an open group', ->
```

Comment:
If I'm joining an open group:

- Redirect to signup page with a group key param
- Upon signup, redirect back to that group in the angular app, logged in as the newly signed up user

PS @HSalmon this is how to protractor test non-angular pages, which might come in handy down the road.